### ISSUE TITLE
Missing exposure bias [EV] exif tag

### ISSUE THREAD

**Issue by pixmin** [darktable-org/darktable#2680]
Hi,

In 2.7 the option to configure EXIF tags shown can be tweaked with this new setting:

Pattern for the image info line

It says "see manual to see all the tags you can use". I can't find any mention of those tags... I grep'ed the source code to find the EXIF_xxx tags, but can't find any "exposure bias" related one... This is super important to see when you intentionally under/over expose a photo using the EV setting on camera, or when do you do bracketing.

I can be read from the photo with exiv2:

```
[gaetan@xps 06. Piana]$ exiv2 IMG_9564.JPG 
File name       : IMG_9564.JPG
[...]
Exposure time   : 1/400 s
Aperture        : F5.6
Exposure bias   : 0 EV
Flash           : No, compulsory
[...]

```

How can it be used?

Thanks

**Comment by junkyardsparkle** [comment]
There are only a few Exif tags imported into darktable's internal metadata, and Exif.Photo.ExposureBiasValue (0x9204) isn't currently one of them. I don't know what the policy is for adding more tags; I suppose this one could be useful for things like auto-applying certain presets...

**Comment by pixmin** [comment]
Thanks @junkyardsparkle , I've tried to amend the code to show it, but I must be missing something... I could use some pointers :)

Attached is the patch with my changes.
[dt_ev.patch.txt](https://github.com/darktable-org/darktable/files/3265725/dt_ev.patch.txt)

**Comment by junkyardsparkle** [comment]
I'm generally afraid of hacking on anything that involves adding fields to the database, being pretty clueless about the implications there... but offhand I would say that you want to handle that value similarly to, for example, focal length, not as a string. The exiftool tag reference is a good general source: http://owl.phy.queensu.ca/~phil/exiftool/TagNames/EXIF.html

**Comment by aurelienpierre** [comment]
Fixed by #4592
